styles/bstats/catppuccin.user.css

```diff
@@ -0,0 +1,653 @@
+/* ==UserStyle==
+@name bStats Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/bstats
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/bstats
+@version 0.1.0
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/bstats/catppuccin.user.css
+@description Soothing pastel theme for bStats
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select flavor "Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green*", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Grey"]
+
+@var checkbox graphUseAccentColor "Graphs Use Accent" 0
+@var select fontMain "Main Font" {
+	"Roboto*":		"Roboto, sans-serif",
+	"Montserrat":	"Montserrat",
+	"Poppins":		"Poppins",
+	"Monocraft":	"Monocraft"
+}
+@var select fontSecondary "Secondary Font" {
+	"Signika*":		"Signika, serif",
+	"Montserrat":	"Montserrat",
+	"Poppins":		"Poppins",
+	"Monocraft":	"Monocraft"
+}
```

Comment:
Although this is nice, we prefer to not change fonts within catppuccin userstyle ports. It would be best if this is removed and you created your own seprate userstyle for this, thanks.